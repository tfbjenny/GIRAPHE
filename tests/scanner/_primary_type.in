void int float string bool node graph edge list hashmap null
