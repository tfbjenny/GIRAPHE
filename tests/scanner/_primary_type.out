INT
FLOAT
STRING
BOOL
NODE
EDGE
GRAPH
LIST
DICT
NULL
